source/exec.tex

```diff
@@ -5672,3 +5690,845 @@
+element of \tcode{args}. Otherwise, \exposid{alloc} is initialized
+with \tcode{allocator_type()}.
+\end{itemdescr}
+
+\begin{itemdecl}
+task get_return_object() noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+A \tcode{task} object whose member \exposid{handle} is
+\tcode{coroutine_handle<promise_type>::\linebreak from_promise(*this)}. % avoid Overfull
+\end{itemdescr}
+
+\begin{itemdecl}
+auto initial_suspend() noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+An awaitable object of unspecified type \iref{expr.await} whose
+member functions arrange for
+
+\begin{itemize}
+\item the calling coroutine to be suspended,
+\item the coroutine to be resumed on an execution agent of the
+execution resource associated with \tcode{SCHED(*this)}.
+\end{itemize}
+\end{itemdescr}
+
+\begin{itemdecl}
+auto final_suspend() noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+An awaitable object of unspecified type \iref{expr.await} whose
+member functions arrange for the completion of the asynchronous
+operation associated with \tcode{STATE(*this)} by invoking:
+
+\begin{itemize}
+\item
+    \tcode{set_error(std::move(RCVR(*this)), std::move(e))}
+    if \tcode{\exposid{errors}.index()} is greater than zero and
+    \tcode{e} is the value held by \exposid{errors}, otherwise
+\item
+    \tcode{set_value(std::move(RCVR(*this)))} if \tcode{is_void<T>} is \tcode{true},
+    and otherwise
+\item
+ \tcode{set_value(std::move(RCVR(*this)), *\exposid{result})}.
+\end{itemize}
+\end{itemdescr}
+
+\begin{itemdecl}
+template<class Err>
+auto yield_value(with_error<Err> err);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\mandates
+\tcode{std::move(err.error)} is convertible to exactly one of the
+\tcode{set_error_t} argument types of \tcode{error_types}. Let
+\tcode{Cerr} be that type.
+
+\pnum
+\returns
+An awaitable object of unspecified type \iref{expr.await} whose
+member functions arrange for the calling coroutine to be suspended
+and then completes the asynchronous operation associated with
+\tcode{STATE(*this)} by invoking \tcode{set_error(std::move(RCVR(*this)),
+Cerr(std::move(err.error)))}.
+\end{itemdescr}
+
+\begin{itemdecl}
+template<@\libconcept{sender}@ Sender>
+auto await_transform(Sender&& sndr) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+If \tcode{same_as<inline_scheduler, scheduler_type>} is \tcode{true}
+returns \tcode{as_awaitable(\linebreak std::forward <Sender>(sndr), % avoid Overfull
+*this)}; otherwise returns
+\tcode{as_awaitable(affine_on(\linebreak std::forward<Sender>(sndr), % avoid Overfull
+SCHED(*this)), *this)}.
+\end{itemdescr}
+
+\begin{itemdecl}
+template<class Sch>
+auto await_transform(change_coroutine_scheduler<Sch> sch) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{returns await_transform(just(exchange(SCHED(*this),
+\linebreak scheduler_type(sch.scheduler))), *this);} % avoid Overfull
```

Comment:
```suggestion
Equivalent to:
\begin{codeblock}
return await_transform(just(exchange(SCHED(*this),  scheduler_type(sch.scheduler))), *this);
\end{codeblock}
```
Note the typo in `returns` (paper has this), but I think this doesn't really need a separate commit.